tests/e2e/specs/api/PhpDevFileAPI.spec.ts

```diff
@@ -0,0 +1,70 @@
+import {DevfilesRegistryHelper} from "../../utils/DevfilesRegistryHelper";
+import {ContainerTerminal, KubernetesCommandLineToolsExecutor} from "../../utils/KubernetesCommandLineToolsExecutor";
+import {ShellExecutor} from "../../utils/ShellExecutor";
+import {DevWorkspaceConfigurationHelper} from "../../utils/DevWorkspaceConfigurationHelper";
+import {DevfileContext} from "@eclipse-che/che-devworkspace-generator/lib/api/devfile-context";
+import {ShellString} from "shelljs";
+import {expect} from "chai";
+import {API_TEST_CONSTANTS} from "../../constants/API_TEST_CONSTANTS";
+import YAML from "yaml";
+import {StringUtil} from "../../utils/StringUtil";
+import {string} from "yaml/dist/schema/common/string";
+
+
+suite('PHP devfile API test', function (): void {
+    const devfilesRegistryHelper: DevfilesRegistryHelper = e2eContainer.get(CLASSES.DevfilesRegistryHelper);
+    const kubernetesCommandLineToolsExecutor: KubernetesCommandLineToolsExecutor = e2eContainer.get(CLASSES.KubernetesCommandLineToolsExecutor);
+    let containerTerminal: ContainerTerminal = e2eContainer.get(CLASSES.ContainerTerminal);
+    let devWorkspaceConfigurationHelper: DevWorkspaceConfigurationHelper;
+    let devfileContext: DevfileContext;
+    let devfileContent: string ='';
+
+    suiteSetup(`Prepare login ${BASE_TEST_CONSTANTS.TEST_ENVIRONMENT}`, async function (): Promise<void> {
+        kubernetesCommandLineToolsExecutor.loginToOcp();
+    });
+
+    test('Create  PHP DevSpace', async function (): Promise<void> {
+        kubernetesCommandLineToolsExecutor.namespace = 'admin-devspaces';
+        devfileContent = devfilesRegistryHelper.getDevfileContent('php');
+        const editorDevfileContent: string = devfilesRegistryHelper.obtainCheDevFileEditorFromCheConfigMap('editors-definitions');
+        kubernetesCommandLineToolsExecutor.workspaceName = YAML.parse(devfileContent).metadata.name;
+
+        devWorkspaceConfigurationHelper = new DevWorkspaceConfigurationHelper({
+            editorContent: editorDevfileContent, devfileContent: devfileContent
+        });
+        devfileContext = await devWorkspaceConfigurationHelper.generateDevfileContext();
+
+        const devWorkspaceConfigurationYamlString: string =
+            devWorkspaceConfigurationHelper.getDevWorkspaceConfigurationYamlAsString(devfileContext);
+        const output: ShellString = kubernetesCommandLineToolsExecutor.applyAndWaitDevWorkspace(devWorkspaceConfigurationYamlString);
+        expect(output.stdout).contains('condition met');
+    });
+
+    test('Check running application', async function (): Promise<void> {
+        const workdir:string = YAML.parse(devfileContent).commands[0].exec.workingDir;
+        const commandLine:string = YAML.parse(devfileContent).commands[0].exec.commandLine;
+        const containerName:string = YAML.parse(devfileContent).commands[0].exec.component;
+        const runCommandInBash:string = `cd ${workdir} && ${commandLine}`;
```

Comment:
It would also be useful to see the actual values of `workdir`  and `commandLine`: in the logs to recognize original values and the command created from them.